examples/grover/1/main.py

```diff
@@ -0,0 +1,39 @@
+from qibo import gates
+from qibo.models.grover import Grover
+from qibo.models import Circuit
+import argparse
+
+def main(qubits):
+    """Create an oracle, find state |11...11> for a number of qubits
+    Args:
+        qubits (int): number of qubits
+
+    Returns:
+        solution (str): found string
+        iterations (int): number of iterations needed
+    """
+    superposition = Circuit(qubits)
+    superposition.add([gates.H(i) for i in range(qubits)])
+
+
+    oracle = Circuit(qubits + 1)
+    oracle.add(gates.X(qubits + 1).controlled_by(*range(qubits)))
+    # Create superoposition circuit: Full superposition over the selected number qubits.
+
+    # Generate and execute Grover class
+    grover = Grover(oracle, superposition_circuit=superposition, number_solutions=1)
+
+    solution, iterations = grover()
+
+    print('The solution is', solution)
+    print('Number of iterations needed:', iterations)
+
+    return solution, iterations
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--qubits", default=5, type=int)
```

Comment:
@AdrianPerezSalinas this example is not working for me, it crashes at line 20.
Could you please rename `qubits` in `nqubits` in order to match the syntax of other examples?

Reply:
Does the problem come from this mismatch? I have tested it and it works

Reply:
How is the `test_examples.py` file run? I cannot see it clearly and I think here is where the problem lies